src/Pages/Result/Result.css

```diff
@@ -0,0 +1,91 @@
+.result-outer-container {
+    height: 37rem;
+    width: 81rem;
+    display: flex;
+    justify-content: center;
+    overflow-y: scroll;
+}
+
+.result-content-container {
+    height: 115rem;
+    width: 35rem;
+    display: flex;
+    justify-content: center;
+    margin-top: 1rem;
+    box-shadow: 1px 1px 5px 1px var(--gainsboro-color);
+    border-radius: 1rem;
+
+}
+
+.result-inner-div {
+    height: 5.5rem;
+    width: 30rem;
+    margin-top: 0.5rem;
+    color: var(--white-color);
+}
+
+.resultOptions {
+    height: 3rem;
+    width: 100%;
+    border-radius: 0.5rem;
+    margin-top: 1rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    color: var(--black-color);
+    font-weight: bold;
+    background-color: var(--gainsboro-color);
+}
+
+.correct_option {
+    height: 3rem;
+    width: 100%;
+    border-radius: 0.5rem;
+    margin-top: 1rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    color: var(--black-color);
+    font-weight: bold;
+    background-color: var(--green-color);
+}
+
+.wrong_option {
+    height: 3rem;
+    width: 100%;
+    border-radius: 0.5rem;
+    margin-top: 1rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    color: var(--black-color);
+    font-weight: bold;
+    background-color: var(--red-color);
+}
```

Comment:
You can keep same properties in one class make a new utility classes for the different ones, say ```bg-red``` and ```bg-green```. This will lower code repetition and improve readability